docker-compose.yml

```diff
@@ -80,6 +80,35 @@ services:
       - homepage.widget.type=radarr
       - homepage.widget.url=http://radarr:7878/radarr
       - homepage.widget.key=${RADARR_API_KEY}
+  jellyseerr:
+    image: fallenbagel/jellyseerr:latest
+    container_name: jellyseerr
+    environment:
+      - LOG_LEVEL=debug
+      - TZ=${TIMEZONE}
+    ports:
+      - 5055:5055
+    volumes:
+      - ./jellyseerr:/app/config
+    restart: always
+    labels:
+      - traefik.enable=true
+      ## HTTP Routers
+      - traefik.http.routers.jellyseerr-rtr.rule=Host(`jellyseerr.${HOSTNAME}`)
```

Comment:
In general, the pattern of this project is to have all the projects in subfolders, could you please change that? same applies for the `homepage.href` label

Reply:
Overseerr unfortunately doesn't support baseurls. I tried for some time trying to get it to work with stripurls middleware and it wouldn't budge.

More info:
https://docs.overseerr.dev/extending-overseerr/reverse-proxy
https://github.com/sct/overseerr/issues/274

In that github issue somebody got it to work doing some messy stuff by adding nginx in front, but not sure its worth it.

Reply:
If it is not supported, then it should definitely be an optional service in my opinion, I'll have a look when I get a chance :) thanks for the changes!